javascript/javascript3/week1/giffy.js

```diff
@@ -0,0 +1,30 @@
+const searchButton = document.querySelector("#myBtn");
+const inputWord = document.querySelector(".user-input")
+const inputNumber = document.querySelector(".input-number");
+const output = document.getElementsByClassName(".result");
+
+searchButton.addEventListener("click", () => {
+    let keyWord = inputWord.value;
+    let getNumber = inputNumber.value;
+
+    if (inputWord.value === "") {
+        alert("Enter a keyword");
+    } else {
+        fetch(
```

Comment:
It almost looks like you've been trying `async/await` (weather app) and Promise-chains (giphy app) to see which one you like best, very good 👍